crates/transaction-pool/Cargo.toml

```diff
@@ -54,6 +54,9 @@ rand = { workspace = true, optional = true }
 paste = { version = "1.0", optional = true }
 proptest = { workspace = true, optional = true }
 
+# for eip-4844
```

Comment:
I don't think we need this

Reply:
My bad, fixed, residual of the experimentation I did.